docs/src/NoncommutativeAlgebra/PBWAlgebras/ideals.md

```diff
@@ -35,13 +35,13 @@ If `I` is an ideal of a PBW-algebra  `A`, then
 
 ```jldoctest
 julia> D, (x, y, dx, dy) = weyl_algebra(QQ, ["x", "y"])
-(Weyl-algebra over Rational field in variables (x, y), PBWAlgElem{QQFieldElem, Singular.n_Q}[x, y, dx, dy])
+(Weyl-algebra over rational field in variables (x, y), PBWAlgElem{QQFieldElem, Singular.n_Q}[x, y, dx, dy])
```

Comment:
Orthogonal to this PR (and not a change request for it): it really also should be `Weyl algebra` and `PBW algebra` (without the `-`)